### PR TITLE
add test case for throwing on non-executables

### PR DESCRIPTION
**Pull request by KnisterPeter**
add a test case for #47 
In master this does not fail anymore, but a test case if it may help to avoid regression

### CODE REVIEW

**Comment by sindresorhus** [comment]
Would you be able to fix the merge conflict?

**Comment by KnisterPeter** [comment]
Sure :)

**Comment by KnisterPeter** [comment]
I think it may be hard to test on windows boxes :(

**Comment by KnisterPeter** [comment]
I will give it some thoughts tomorrow.

File: test.js [hunk 1]
```diff
@@ -166,6 +167,13 @@ test(`use relative path with '..' chars`, async t => {
 	t.is(stdout, 'foo');
 });
 
+test('execa() throws if running non executable', async t => {
+	// windows systems do not know executable-flags
+	if (os.platform() !== 'win32') {
```
Comment (sindresorhus) [inline test.js]: Better to wrap the whole test in that if statement. Also no need to import `os`. Just use `process.platform`.

**Pull request merged**